### PR TITLE
Travis test also for python 3

### PR DESCRIPTION
**Pull request by verschoof**
Test the code also for the new python version 3.7
And drop support for Ansible 2.4 and lower to improve test speed.

This is a more a PR suggestion, would like to know your opinion 😄

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,9 @@ sudo: required
 dist: trusty
 
 language: python
-python: "2.7"
+python:
+  - "2.7"
+  - "3.7"
```
Comment (tersmitten) [inline .travis.yml]: https://docs.travis-ci.com/user/languages/python/

**Comment by verschoof** [comment]
ping @tersmitten :)

**Comment by tersmitten** [comment]
I hope #6 solves you need.

**Comment by verschoof** [comment]
@tersmitten Yea, looks good to me, thanks! :) Weird thing is, in the PR your tests are failing, but after the merge the pass again.. oh well 😃

**Comment by tersmitten** [comment]
That's because I merged it before all the tests were complete.

Met vriendelijke groeten,
Mischa ter Smitten
Software & system architect
mtersmitten@oefenweb.nl
[image: Oefenweb.nl] <http://www.oefenweb.nl/>webapplicaties voor adaptief
onderwijs
 Facebook <https://www.facebook.com/Oefenweb> Twitter
<https://twitter.com/mtersmitten> LinkedIn
<https://www.linkedin.com/in/mtersmitten>

**Pull request closed without merging**